### PR TITLE
Extend expiring switches

### PR DESCRIPTION
**Pull request by ioannakok**
## What does this change?
Extend expiring switches

### CODE REVIEW

File: common/app/conf/switches/ABTestSwitches.scala [hunk 1]
```diff
@@ -61,7 +61,7 @@ trait ABTestSwitches {
     "Remove the initialisation of Prebid and A9 in the Canada region",
```
Comment (Georges-GNM) [inline common/app/conf/switches/ABTestSwitches.scala:61]: Looks like @domlander is intending to remove this switch? https://github.com/guardian/frontend/pull/25867

**Review by bryophyta — APPROVED** [review]
The code looks good to me!

Out of interest (I haven't been involved with this before) is there a process for checking sell by dates with experiment owners?

File: common/app/conf/switches/ABTestSwitches.scala (hunk 1 above)
Comment (ioannakok) [reply common/app/conf/switches/ABTestSwitches.scala:61]: Good call! I will revert my change for this PR.

**Comment by ioannakok** [comment]
> 

I think the idea is the owner extends the switches but I'm not sure the emails work as expected so sometimes the owners don't get notified. I will add a rota ticket for us to check! 

I created this PR to keep main green because there's already one that expired yesterday and thought I'd extend the rest. I only keep `ab-billboards-in-merch` and `header-top-bar-search-capi` and check with the owners for the rest!

**Review by paulmr — APPROVED** [review]

**Review by chrislomaxjones — APPROVED** [review]
Looks good from a commercial perspective 👍

**Review by simonbyford — APPROVED** [review]
Thanks for this!

**Review by domlander — APPROVED** [review]

**Comment by bryophyta** [comment]
> I think the idea is the owner extends the switches but I'm not sure the emails work as expected so sometimes the owners don't get notified. I will add a rota ticket for us to check.
> 
> I created this PR to keep main green because there's already one that expired yesterday and thought I'd extend the rest. I'll check with the owners before merging!

very helpful explanation, thanks!

**Pull request merged**